lib/typescript/render/components/Carousel/index.module.scss

```diff
@@ -5,7 +5,7 @@
   display: flex;
   align-items: flex-end;
   flex-direction: column;
-  margin-top: 8px;
+  margin-top: 5px;
   overflow: hidden;
   position: relative;
```

Comment:
the different render components had different values for the _margin-top_ (which displayed weirdly in the modal template) so I've unified everything at 5px (following the Text components)